nixos/tests/web-apps/gotosocial.nix

```diff
@@ -1,7 +1,7 @@
-{ lib, ... }:
+{ pkgs, lib, ... }:
```

Comment:
```suggestion
{ lib, gotosocial, ... }:
```

Reply:
@drupol I think I'm going to roll this test back. `gotosocial` isn't bound / passed into this function. I'm not sure this test has high value anyways: it feels like a bit of a smell to pass in `pkgs` to this test function.

```
error:
       … while evaluating the attribute 'drvPath'

         at /nix/store/k427qnlyr0ibh3qdc16zncxz1lxry5z7-source/lib/customisation.nix:268:7:

          267|     in commonAttrs // {
          268|       drvPath = assert condition; drv.drvPath;
             |       ^
          269|       outPath = assert condition; drv.outPath;

       … while calling the 'derivationStrict' builtin

         at /builtin/derivation.nix:9:12: (source not available)

       (stack trace truncated; use '--show-trace' to show the full trace)

       error: attribute 'gotosocial' missing

       at /nix/store/k427qnlyr0ibh3qdc16zncxz1lxry5z7-source/lib/modules.nix:508:28:

          507|         builtins.addErrorContext (context name)
          508|           (args.${name} or config._module.args.${name})
             |                            ^
          509|       ) (lib.functionArgs f);
```